backend/s3/s3.go

```diff
@@ -1536,7 +1536,16 @@ isn't set then "acl" is used instead.`,
 			}},
 		}, {
 			Name:     "sse_customer_key",
-			Help:     "If using SSE-C you must provide the secret encryption key used to encrypt/decrypt your data.",
+			Help:     "If using SSE-C you must provide the secret encryption key used to encrypt/decrypt your data. Alternatively you can provide sse_customer_key_base64.",
```

Comment:
The `Help:` is used to make to make the docs and also the help for the flag.

Only the first line is used for the flag help text (try `rclone help flags sse`)

The other lines should be separated by a blank line and wrapped to 80 chars.

So I think this needs to be something like this. Note that we usually
refer to options by their flag name in the docs.

```go
			Help:     `Secret encryption key for SSE-C

If using SSE-C you must provide the secret encryption key used to
encrypt/decrypt your data. Alternatively you can provide
--sse-customer-key-base64.`,

```